.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create an issue to track a bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Description
+
+A clear and concise description of what the bug is.
+
+- Does it occur on any specific browsers or breakpoints?
+- Remember to assign this to one of our GitHub project boards and to a milestone if applicable
+
+## Steps To Reproduce
+
+Steps to reproduce the behaviour:
+1. Go to '…'
+2. Click on '…'
+3. Scroll down to '…'
+4. See error
+
+## Screenshots
+
+*Delete if unnecessary.*
+
+| Screenshot One | Screenshot Two |
```

Comment:
This follow the before and after pattern we use for PRs but I think for bugs there's perhaps less need for this?

Reply:
Yeah it's a good point, I wasn't sure on this. I wanted make it easy/prompt people to include screenshots.

I think having a table for screenshots is useful because a) it's nice to have that already there for you rather than typing out all the `| - |` yourself, and b) it constrains the images to prevent them going full width and requiring a lot of scrolling.

That said, the structure of the table is perhaps less consistent from issue to issue as "Before"/"After" for PRs. My thought was I'll just include a very basic structure and people can adapt it to their needs.

What do you think?